### ISSUE TITLE
Failed to retrieve library!

### ISSUE THREAD

**Issue by 0x73am43l** [sharkwouter/minigalaxy#320]
I get this warning when I open the app and it does not show the game list 
![w](https://user-images.githubusercontent.com/13575387/115345001-662f4280-a1c3-11eb-8147-d96aa0ef0fbf.png)

**Comment by sharkwouter** [comment]
How many games for Linux do you have?

**Comment by makson96** [comment]
This does look like duplicate of #271
Could you reproduce this issue with master branch?

**Comment by thexavier666** [comment]
I am having the same issue @makson96 

```
minigalaxy --version
1.0.2
```

Resolved only after adding linux-compatible titles to account.
There should be a different error "No linux games present in account"

**Comment by makson96** [comment]
@thexavier666 this issue is already addressed in the master branch.
I would like @0x73am43l to confirm it is the same issue before closing it.

**Comment by sharkwouter** [comment]
This should be fixed in version 1.1.0
